docs/guides/build_a_simple_pipeline.md

```diff
@@ -44,7 +45,7 @@ Fondant offers three distinct component types:
 - **Generic components**: They provide the business logic but may require adjustments to the component spec.
 - **Custom components**: The component implementation is user-dependent.
 
-If you want to learn more about components, you can check out the [components documentation](component.md).
+If you want to learn more about components, you can check out the [components documentation](/components/components).
```

Comment:
```suggestion
If you want to learn more about components, you can check out the [components documentation](/components/components.md).
```

Reply:
From my side it doesn't work when I add the .md. The only way is with /components/components. Does it work for you?

Reply:
Yeah indeed, both don't work for me. I think `components/components` will work (without the leading slash)

Reply:
`components/components` doesn't work...

Reply:
Where did you test it?

Reply:
On the docs website. I run mkdocs serve and check it locally

Reply:
It doesn't work on the built website, since it contains a base path (eg. `en/latest`).

The correct way to link this would be with a full relative path and with a `.md` extension ([see docs](https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages)). In this case: `../components/components.md`

Can you update this everywhere except for the main readme, where we need the absolute urls?

Reply:
Oh okay understood! So if I understood well, concerning links, checking if they work locally is not relevant. Better push and check the built website. (or check here on github but I don't think there's a way to check before pushing, or is there a way?)

Reply:
Yeah indeed, unless we can make it add a base_path locally as well